### PR TITLE
"Invoice" to "Order"

### PR DESCRIPTION
**Pull request by torvista**
as per 
https://www.zen-cart.com/showthread.php?226687-Use-of-quot-order-quot-instead-of-quot-invoice-quot&p=1368580#post1368580

### CODE REVIEW

**Comment by scottcwilson** [comment]
The button says "Invoice" and the labels on the page should be consistent.  This should not be merged.

File: includes/languages/english/checkout_process.php [hunk 1]
```diff
@@ -14,7 +14,7 @@
 define('EMAIL_DETAILS_FOLLOW','The following are the details of your order.');
 define('EMAIL_TEXT_ORDER_NUMBER', 'Order Number:');
 define('EMAIL_TEXT_INVOICE_URL', 'Detailed Invoice:');
```
Comment (drbyte) [inline includes/languages/english/checkout_process.php:16]: Was this intentionally left unchanged?

File: admin/includes/languages/english/invoice.php [hunk 2]
```diff
@@ -29,5 +29,5 @@
 define('ENTRY_SHIPPING', 'Shipping:');
 define('ENTRY_DATE_PURCHASED', 'Date Ordered:');
 
-define('ENTRY_ORDER_ID','Invoice No. ');
+define('ENTRY_ORDER_ID','Order No. ');
```
Comment (drbyte) [inline admin/includes/languages/english/invoice.php:32]: Was this same change in packingslip.php intentionally skipped?

**Comment by zcwilt** [comment]
Guess I should add as well, that strictly speaking an Order id and an Invoice/Receipt number should really be two separate entities and while Zen Cart doesn't support that, it's something we should look to do.

**Comment by torvista** [comment]
As I have mentioned ad nauseum, I think an "invoice" is a legal term, requiring specific items to qualify as an "invoice" legally, per-country definitions. Even "ticket" in spain is classed as a special type of anaonymous invoice with its own specification of what it must include to be legal.

**Pull request merged**